ml/src/test/scala/org/apache/spark/ml/spark/models/H2OMOJOPipelineModelTest.scala

```diff
@@ -106,10 +108,11 @@ class H2OMOJOPipelineModelTest extends FunSuite with SparkTestContext {
     // Test data
     val df = spark.read.option("header", "true").csv("examples/smalldata/prostate/prostate.csv")
     // Test mojo
+    val mojoSettings = H2OMOJOSettings(namedMojoOutputColumns = false)
```

Comment:
should be `namedMojoOutputColumns = false`

Reply:
Good catch, thanks!

Reply:
I missed that during review, sorry

Reply:
There is no reason to apologize. It was my mistake ;-)